server/Dockerfile.application

```diff
@@ -0,0 +1,14 @@
+ARG image_tag=latest
+
+FROM liberoadmin/reviewer_server_typescript:${image_tag}
```

Comment:
the commit messages say `server` is meant to use a multi-stage build but at the moment every image is extending the previous one rather than being a new stage. This would be the multi-stage version:

```
FROM liberoadmin/reviewer_server_typescript:${image_tag} AS typescript
 FROM node:10-alpine

COPY --from=typescript /files/that/I/need ./
```
I'm not sure if this is useful, but it would for example throw away the Yarn cache if there is any, or any intermediate product of `yarn build` that is not being copied over.